src/status_im2/contexts/wallet/common/sheets/network_preferences/view.cljs

```diff
@@ -1,70 +1,92 @@
 (ns status-im2.contexts.wallet.common.sheets.network-preferences.view
-  (:require [quo.core :as quo]
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
             [quo.foundations.colors :as colors]
             [quo.foundations.resources :as resources]
             [quo.theme :as quo.theme]
+            [reagent.core :as reagent]
             [status-im2.contexts.wallet.common.sheets.network-preferences.style :as style]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
-(defn- mainnet
-  [account-color]
-  [{:title        "Mainnet"
-    :image        :icon-avatar
-    :image-props  {:icon (resources/get-network :ethereum)
-                   :size :size-20}
-    :action       :selector
-    :action-props {:type                :checkbox
-                   :customization-color account-color}}])
 
-(defn- networks-list
-  [account-color]
-  [{:title        "Optimism"
-    :image        :icon-avatar
-    :image-props  {:icon (resources/get-network :optimism)
-                   :size :size-20}
-    :action       :selector
-    :action-props {:type                :checkbox
-                   :customization-color account-color}}
-   {:title        "Arbitrum"
-    :image        :icon-avatar
-    :image-props  {:icon (resources/get-network :arbitrum)
-                   :size :size-20}
-    :action       :selector
-    :action-props {:type                :checkbox
-                   :customization-color account-color}}])
+(defn- make-network-item
+  [{:keys [network-name] :as _network}
+   {:keys [color on-change network-preferences] :as _options}]
```

Comment:
Could you please elaborate a bit on using undersсore for naming?

Reply:
@vkjr 
In Clojure it's the same as just naming a var `_`. It is used to name a var an be ignored in the code.
Linters like clj-kondo or the one built-in in Cursive/intelliJ understand this thing and don't throw any warning about it.

I also use them to make the code easier to understand, when we receive maps and just want to name the whole map, e.g.
```clojure
(defn foo
  [{:keys [username] :as _user}
   {:keys [name last-name] :as _person}
   {:keys [number] :as _address}]
  ,,,)
```
for the `_address` map, we just extract the `:number` keyword, if it were just `{:keys [number]}` it may be a little cofusing, when we add `_address` IMO is clearer that we expect an address map.

Other uses I find it useful, is in ignored args in map's or reduce's fns:
e.g.
```clojure
(reduce-kv (fn [acc _chain-id chain-details]
             (conj acc chain-details))
           []
           {10 {:data "data"}
            20 {:data "data"}
            30 {:data "data"}})
```
Here we don't care about the chain-id, but instead of just ignoring with `_`, we call it `_chain-id`

Reply:
Thanks for the explanation!